### ISSUE TITLE
Investigate Community Code for IIIF Manifest Parsing

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#3319]
### Description
Other pieces of software and community efforts have been undertaken to parse IIIF Presentation 3.0 manifests. Should we use one of these in our own player instead of writing our own parsing logic? This is an issue to investigate what is being done elsewhere and see what we could use or adopt.

### Done Looks Like
- Look at [Manifesto](https://github.com/IIIF-Commons/manifesto) (Universal Viewer parsing code) and evaluate that in relation to updating our own code for parsing
- Look at other code/projects of interest

### Notes
Timeboxed to a 2—looking around and seeing where we're at.

**Comment by adamjarling** [comment]
@mcwhitaker Is there still some value in writing our own IIIF 3.0 parsing logic?   Or is it now ok to pursue an established parsing solution which may exist?

**Comment by davidschober** [comment]
I would prefer to standardize on a community parsing solution. I see the value in a player customized to Avalon's needs, but I think we would gain a lot from using a standard solution.

**Comment by adamjarling** [comment]
@cjcolvar  I pulled down https://github.com/ProjectMirador/mirador, to see how they're using `manifesto.js`, and it's starting to make sense now.  I'll keep playing with it, but I think we should absolutely use `manifesto.js` to pull data out of our IIIF manifests, provided it works out as it is now in some initial testing.

**Comment by mcwhitaker** [comment]
I asked in the IIIF general slack channel about the requirements for implementers and was pointed to this:

_In order to be considered ready for final review, new features must have two open-source server-side implementations, at least one of which should be in production. New features must also have at least one open-source client-side implementation, which may be a proof-of-concept._

Our player does not implement all the features of Prezi 3, so based on my understanding it cannot be counted towards readiness (unless there is another implementation that implements what we don't and doesn't implement what we do... which would be complicated to consider).

In summary, I think we can reuse that parser...
